modules/module_manager/test/modulemanager_test.php

```diff
@@ -0,0 +1,93 @@
+<?php declare(strict_types=1);
```

Comment:
The test file name should be module_managerTest.php

Reply:
The filename was based on Statistics_Test, why does that have the underscore but this one shouldn't? And how can I tell if it's being run in Travis or not?

Reply:
You added two new test functions, but the total test number didn't change.

Reply:
> Test file name should follow the rule as module name (module_manager not modulemanager) with Test or _Test.

Do you know where this is handled? I remember seeing it once but I can't find it now

Reply:
It should be default behavior of the phpunit XML configuration.

Reply:
I don't see where that's happening, the phpunit xml configuration just specifies the list of test directories, but I've renamed the file.